### PR TITLE
Add "promptEnv" setting to toggle prompting for environment vars

### PR DESCRIPTION
**Pull request by mxsdev**
Part of #430

### CODE REVIEW

File: src/extension/executors/utils.ts [hunk 1]
```diff
@@ -194,11 +195,12 @@ export function getShellPath(execKey?: string): string|undefined {
  */
 export async function parseCommandSeq(
   cellText: string,
+  promptForEnv = true,
```
Comment (christian-bromann) [inline src/extension/executors/utils.ts:198]: See here is the default value used again

File: src/extension/executors/utils.ts [hunk 2]
```diff
@@ -109,12 +110,12 @@ export function getCommandExportExtractMatches(
  * @param exec NotebookCellExecution
  * @returns cell text if all operation to retrieve the cell text could be executed, undefined otherwise
  */
-export async function retrieveShellCommand (exec: NotebookCellExecution) {
+export async function retrieveShellCommand (exec: NotebookCellExecution, promptForEnv = true) {
```
[the remark this answers is not in the document]
Comment (mxsdev) [reply src/extension/executors/utils.ts:113]: Yeah, will do

**Comment by mxsdev** [comment]
CI keeps failing on build :\ not sure why

**Review by christian-bromann — APPROVED** [review]
LGTM 👍 

Let's not forget to update our docs.

**Comment by mxsdev** [comment]
@christian-bromann 

Created stateful/runme.dev#140, let me know if that is sufficient to merge

**Pull request merged**